ui/src/routes/runs/$runId/index.tsx

```diff
@@ -52,7 +55,22 @@ export const Route = createFileRoute('/runs/$runId/')({
   },
   errorComponent: ({ error }) => {
     if (error instanceof NotFoundError) {
-      return <>There is no run with ID {error.message}.</>;
+      return (
+        <Card className='mx-auto w-full max-w-4xl'>
+          <CardHeader className='flex flex-row justify-items-center p-4'>
+            <div className='flex gap-2'>
+              <AlertCircle className='size-12' />
+              <h2 className='place-content-center text-3xl font-bold'>
+                No result!
```

Comment:
Mabye use "Resource not found" instead, without the "!" (the icon in front already has one).